packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -112,11 +113,17 @@ class Tooltip extends StatefulWidget {
     this.waitDuration,
     this.showDuration,
     this.child,
-  }) : assert(message != null),
-       super(key: key);
+  })  : assert(
+          tooltip != null || message != null,
+          'A non-null message or tooltip must be provided to a Tooltip Widget',
+        ),
+        super(key: key);
+
+  /// The widget to display in the tooltip.
```

Comment:
This doc should also talk about "message" and explain the difference.

Reply:
message can be replaced with the semantic label property like Text Widget or any other widgets. In such a way, we can make it optional for developers so that they can provide it if they wish to use it for accessibility.